app/controllers/presentations_controller.rb

```diff
@@ -1,8 +1,16 @@
 class PresentationsController < ApplicationController
   def index
-    @presentations = Presentation.joins(:event)
-                                 .merge(Event.order(date: :desc))
-                                 .page(params[:page])
+    if params[:presentation] && params[:presentation][:person_id].present?
```

Comment:
I would also take a look at the methods ruby hash probides you for doing this.

#fetch and #dig